qemu/tests/cfg/memhp_threads.cfg

```diff
@@ -0,0 +1,29 @@
+- memhp_threads:
+    type = memhp_threads
+    required_qemu = [5.0, )
+    paused_after_start_vm = yes
+    mem_fixed = 4096
+    slots_mem = 4
+    maxmem_mem = 32G
+    smp = 4
+    guest_numa_nodes = "node0"
+    mem_devs = mem0
+    use_mem_mem0 = "no"
+    size_mem_mem0 = 4096M
+    backend_mem_mem0 = memory-backend-ram
+    numa_memdev_node0 = mem-mem0
```

Comment:
As there's default numa node on ppc so I doubt if we need to add it or not. If there's no relevant with numa, it's acceptable. Thanks.

Reply:
It won't hurt to specify a numa node explicitly.